Game/Resources/Shaders/VFX/trail.fs

```diff
@@ -6,12 +6,26 @@ layout (location = 2) out vec4 PostProcessFilter;
 
 uniform sampler2D tex;
 uniform vec4 color;
+uniform vec4 color_blend;
 uniform float emissive_intensity;
+uniform float percentage;
+uniform float smooth_step;
 
 void main()
 {
-	vec4 texture_color = texture(tex, texCoord) * color * emissive_intensity;
-	FragColor =  texture_color;
+	vec2 t = texCoord;
+	float f = fract(t.s);
+	float s_step = smoothstep(percentage, percentage + smooth_step, f);
```

Comment:
Try to use better variable names here.

Reply:
oki..ill change it to smth better and recommit